.env.example

```diff
@@ -104,6 +104,10 @@ PUSHER_SECRET=
 
 # GITHUB_TOKEN=
 
+# GitHub client for users to associate their GitHub accounts
```

Comment:
as I learned recently, this should probably also mention where to point the callback url (either here or in setup documentation)